models/testdata/text-file-without-vars.txt

```diff
@@ -0,0 +1,11 @@
+There are no variables in this file.
```

Comment:
Honestly, this is probably all you need for this test - no need for the rest of the long file. 😄

Reply:
Done 😅